adopt-a-dog/blog.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <title>Adopt A Dog</title>
+    </head>
+    <body>
+      <div class="header">
+        <img src="images/christopher-harris-57366-unsplash.jpg" alt="Dog header picture" width="100%" height="5%">
+        <img src="adoptadog-logo.svg" alt="Adopt A Dog logo" width="25%">
+        <h2>Seattle's premiere dog adoption agency</h2>
+      </div>
+      <nav class="site-nav">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dog.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+      <div class="blog">
+        <h2>Adoptadog Blog</h2>
+          <div class="blog-preview">
+            <img src="images/patrick-hendry-221863-unsplash.jpg" alt="dog picture" width="20%">
```

Comment:
The HTML validator doesn't like these values. We can use percents in css, but apparently the validator doesn't like them. Make sure to yank these attributes out when you add a CSS file. I won't ding you on the validator as I encouraged you all to use percentages, but the validator did yell at me.